### ISSUE TITLE
Style the Component TodoPage.js

### ISSUE THREAD

**Issue by Shadow-rR** [Shadow-rR/TaskApp#2]
Use CSS modules to style the TodoPage.js component. The styles have to be mobile-first.

**Comment by shub78910** [comment]
Hey, can i pick this up?

**Comment by Shadow-rR** [comment]
Yes sure...do attach a screenshot of the page in the pr...and if you are stuck anywhere feel free to ask

> Hey, can i pick this up?

Yes sure. Attach a screenshot of the page in the pr and feel free to ask if you are stuck anywhere

**Comment by Shadow-rR** [comment]
Did you run npm start after this command. I dont think this error should cause any trouble loading the app... Just try npm start from the project dir...
